app/react/App/scss/modules/_search.scss

```diff
@@ -379,19 +379,19 @@
   }
 }
 
-.search__filter--date {
-  .fa {
-    float: left;
-    width: 5%;
-    text-align: center;
-    line-height: 23px;
-  }
-  input {
-    float: left;
-    width: 46.5%;
-    margin-left: 1%;
-  }
-}
+// .search__filter--date {
+//   .fa {
+//     float: left;
+//     width: 5%;
+//     text-align: center;
+//     line-height: 23px;
+//   }
+//   input {
+//     float: left;
+//     width: 46.5%;
+//     margin-left: 1%;
+//   }
+// }
```

Comment:
Is this really meant to be kept for some reason? We really try not to have commented text.